package.json

```diff
@@ -26,7 +26,7 @@
     ]
   },
   "dependencies": {
-    "@expo/vector-icons": "10.0.0",
+    "@expo/vector-icons": "^10.0.0",
```

Comment:
As per https://github.com/NMF-earth/nmf-app/blob/master/contributing.md#npm-packages, you might not use `^` or `~` or other semver chars. Could we freeze it to `10.0.2`

Reply:
Oh yeah good catch. I know it specifically says that this way we won’t have to use the lock file, but that’s the purpose of having a lock file. But no matter, I can get that changed tomorrow.

Reply:
Fixed this. And it seems like 10.0.2 was still not quite new enough so setting it to 10.0.6 seemed to do it.